examples/proc_FID_nutation.py

```diff
@@ -35,16 +37,28 @@
     fl.next("Raw Data with averaged scans", fig=fig)
     fig.suptitle("FID Nutation %s" % sys.argv[2])
     # }}}
-    # {{{ Apply overall zeroth order correction
-    s.ift("t2")
+    # {{{ Apply overall zeroth order correction, select the pathway, and apply a rudimentary alignment
     s /= prscr.zeroth_order_ph(
-        prscr.select_pathway(s["t2":0], s.get_prop("coherence_pathway"))
+        prscr.select_pathway(
+            s["t2":signal_range].sum("t2"), s.get_prop("coherence_pathway")
+        )
     )
+    s = prscr.select_pathway(
+        s["t2":signal_range], s.get_prop("coherence_pathway")
+    )
+    # {{{ determine shift for rough alignment, but don't use this yet, because
+    #     I want to see the signal resonance frequency
+    s.ift("t2")
+    shift = s * np.exp(-(pi**2) * s.fromaxis("t2") ** 2 * (2 * 50**2))
+    shift.ft("t2")
+    shift = shift["t2":signal_range].real.run(abs).argmax("t2")
+    shift.set_error(None)
+    # interestingly, if we were correcting here, we would multiply by the phase
+    # here to perform the shift, which would be efficient!
     s.ft("t2")
+    # }}}
```

Comment:
just to improve robustness of integral

Reply:
you moved this to rough_table_integrals which I like